vendor/github.com/alkiranet/alkira-client-go/alkira/internet_applications.go

```diff
@@ -39,6 +39,7 @@ type InternetApplication struct {
 	SnatIpv4Ranges                []*InternetApplicationSnatIpv4 `json:"snatIpv4Ranges,omitempty"`
 	Size                          string                         `json:"size"`
 	Targets                       []InternetApplicationTargets   `json:"targets,omitempty"`
+	IlbCredentialId               string                         `json:"ilbCredentialId"`
```

Comment:
Please make this change in alkira-client-go. This is not the proper way to do it, just to make it easier for testing.

Reply:
Also new optional field should be always `omitempty` to ensure backward compatibility.